README.md

```diff
@@ -11,3 +11,9 @@ which will get Upstart out of the confused state.
 When “`status <jobname>`” says “`<jobname> stop/killed, process 12345`” and
 there’s no such process, run “`workaround-upstart-snafu 12345`” and wait until
 it exits.
+
+To speed things up (i.e. to limit the PID space this script covers), issue the following command:
+
+    echo $N | sudo tee /proc/sys/kernel/pid_max
```

Comment:
Cool.  This seems like an obscure way to edit the pid_max, though.  What about just `sudo echo ...` or `sysctl -w kernel.pid_max=$N` (see http://panosgeorgiadis.com/how-to-change-kernel-max-pid-number)

Also, we should make it clearer that this affects your system permanently and needs to be undone afterward.  in fact, it seems like a lot of work to go to, really... might as well just run the thing and wait! :) Perhaps we should put this under a heading of "caveats" and say "if the system's `pid_max` happens to be very high (the default is 32,768 - you can check yours with `sysctl kernel.pid_max`) this process may take some time.  You may be able to speed it up by temporarily reducing your pid_max to just above the highest active PID:

Find the largest current PID: `ps -axo pid | tail -n 1`
Set the max PID `sysctl -w kernel.pid_max=$N`

(where `N` is some number a few hundred greater than the largest current PID)

Don't forget to set it back to the original value when you're done (unless you want to permanently reduce your maximum PID).